src/main/kotlin/week1/calculator/Operator.kt

```diff
@@ -0,0 +1,24 @@
+package week1.calculator
+
+enum class Operator(
+    val value: String,
+    private val executor: OperatorExecutor,
```

Comment:
지금처럼 별도의 SAM을 정의해줄 수도 있지만 아래와 같이 직접 타입을 명시할 수도 있을 것 같아요! 😃

```suggestion
    private val executor: (Int, Int) -> Int,
```